src/main/java/com/arpnetworking/clusteraggregator/GuiceModule.java

```diff
@@ -112,7 +117,15 @@ protected void configure() {
     @Named("akka-config")
     @SuppressFBWarnings("UPM_UNCALLED_PRIVATE_METHOD") // Invoked reflectively by Guice
     private Config provideAkkaConfig() {
-        return ConfigFactory.parseMap(_configuration.getAkkaConfiguration(), _configuration.toString());
+        try {
+            final String akkaJsonConfig = OBJECT_MAPPER.writeValueAsString(_configuration.getAkkaConfiguration());
```

Comment:
This is necessary because the keys contain periods which when transforming from a map are considered compound path elements. This forces the keys to be considered single path elements even with periods.